depends/patches/boost/fix-win-wake-from-sleep.patch

```diff
@@ -0,0 +1,31 @@
+--- old/libs/thread/src/win32/thread.cpp
++++ new/libs/thread/src/win32/thread.cpp
+@@ -645,7 +645,7 @@
+                     } Detailed;
+                 } Reason;
+             } REASON_CONTEXT, *PREASON_CONTEXT;
+-            static REASON_CONTEXT default_reason_context={0/*POWER_REQUEST_CONTEXT_VERSION*/, 0x00000001/*POWER_REQUEST_CONTEXT_SIMPLE_STRING*/, (LPWSTR)L"generic"};
++            //static REASON_CONTEXT default_reason_context={0/*POWER_REQUEST_CONTEXT_VERSION*/, 0x00000001/*POWER_REQUEST_CONTEXT_SIMPLE_STRING*/, (LPWSTR)L"generic"};
```

Comment:
Jup, this patch is from https://github.com/boostorg/thread/commit/9ac736a4c4214cdb01caaa84a1121b3846dc87b4